app/src/main/java/edu/wpi/first/shuffleboard/app/Main.java

```diff
@@ -10,11 +13,22 @@
  */
 @SuppressWarnings("PMD.UseUtilityClass") // Nope.
 public final class Main {
+  private static final Logger logger = Logger.getLogger(Main.class.getName());
+
   @SuppressWarnings("JavadocMethod")
   public static void main(String[] args) {
     // JavaFX 11+ uses GTK3 by default, and has problems on some display servers
     // This flag forces JavaFX to use GTK2
     System.setProperty("jdk.gtk.version", "2");
+
+    Thread
+        .setDefaultUncaughtExceptionHandler((thread, throwable) -> {
+          logger.log(
+              Level.SEVERE,
+              "Uncaught exception on thread `" + thread.getName() + "`.",
```

Comment:
Why the backticks here?

Reply:
Also, this is always on the main thread so there's no need to parameterize it in the message.

Reply:
You don't ever spawn other threads?

Reply:
We do, but this handler is only used for the main thread